packages/astro/src/assets/index.ts

```diff
@@ -1,5 +1,21 @@
+import type { ImageServiceConfig } from '../@types/astro.js';
+
 export { getConfiguredImageService, getImage } from './internal.js';
 export { baseService, isLocalService } from './services/service.js';
 export { type LocalImageProps, type RemoteImageProps } from './types.js';
 export { emitESMImage } from './utils/emitAsset.js';
 export { imageMetadata } from './utils/metadata.js';
+
+export function getSharpImageService(): ImageServiceConfig {
+	return {
+		entrypoint: 'astro/assets/services/sharp',
+		config: {},
+	};
+}
+
+export function getSquooshImageService(): ImageServiceConfig {
```

Comment:
I'd maybe lean into removing the `get` so it's only `sharpImageService` and `squooshImageService`, as I'd imagine third-party image service would've omitted the `get` too in their API, e.g. `cloudinaryImageService`. But curious to hear other opinions too.